teekkari.py

```diff
@@ -285,5 +285,5 @@ def messageHandler(self, bot, update):
                 self.getTEK(bot, update)
             elif re.match(r'.*[tT]uni.*', msg.text):
                 self.getTUNI(bot, update)
-            elif 'nakuttaa' in msg.text.lower():
+            elif 'nakuttaa' in msg.text.lower() or 'CAADBAADMQADsxUiC1dIx__vVyK5FgQ' == msg.sticker.file_id:
```

Comment:
`msg` doesn't always have `msg.sticker`, and this might crash the bot. Add a null check.